src/helpers/server.ts

```diff
@@ -44,9 +44,9 @@ function handleClientData(request: CommandRequest): boolean {
         case Handlers.REDO:
             undoRedoHandler(request.data, 'redo');
             return true;
-        case Handlers.SHOW_IN_IDE:
-            showInIdeHandler(request.data);
-            return true;
+        // case Handlers.SHOW_IN_IDE:
```

Comment:
Looks good, small note that we could potentially remove the code, as we can always revert, or get it back from the git history :).

Also not sure if now that we `return false;`, if it can case any problem.

Reply:
`return false` is in case handler is not found so rest can response with 400 bad request.

Reply:
okey, so we do not want to respond 400, just want to do nothing now if we get this command. Got it.